ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -399,6 +400,23 @@ def ocs_install_verification(
             f"The failure domain type on LSO cluster with {zone_num} zones should be "
             f"'host' and not {failure_domain}."
         )
+    verify_noobaa_console()
+
+
+def verify_noobaa_console():
```

Comment:
I suggest we create a class that verifies a page and other child classes that verify specific pages, such as noobaa console